src/ui/components/DevTools.tsx

```diff
@@ -104,7 +104,7 @@ function Body() {
         <Toolbar />
         <PanelGroup autoSaveId="DevTools-horizontal" className="split-box" direction="horizontal">
           <Panel
-            className="flex=1 flex h-full overflow-hidden"
```

Comment:
Huh "flex=1" looks like it was a bad style to begin with.

Reply:
Yep that isn't a valid tailwind class